awscli/examples/cloudwatch/put-metric-stream.rst

```diff
@@ -0,0 +1,17 @@
+**To create a metric stream**
+
+The following ``put-metric-stream`` example creates a metric stream named ``QuickFull-GuaFb`` in the specified account. ::
+
+    aws cloudwatch put-metric-stream \
+        --name QuickFull-GuaFbs \
+        --firehose-arn arn:aws:firehose:us-east-1:123456789012:deliverystream/MetricStreams-QuickFull-GuaFbs-WnySbECG \
+        --role-arn arn:aws:iam::123456789012:role/service-role/MetricStreams-FirehosePutRecords-JN10W9B3 \
+        --output-format json \
+        --no-include-linked-accounts-metrics
+
+Output::
```

Comment:
Missing a newline here:

```suggestion
Output::

```

Reply:
I think this is causing a couple of the tests failures. Let's resolve this one and I'll check the tests again.

Reply:
Added new line in the last commit.